src/main/java/medicaldoctor/controllers/navbar/NavigationBarController.java

```diff
@@ -72,103 +81,123 @@
     private Button buttonLiverFunction;
     
     @FXML
-    void patientRegistration(ActionEvent event){
-        
+    void patientRegistration(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.PATIENT_REGISTRATION);
+        AppSession.cm.showScreen(LookUp.PATIENT_REGISTRATION);
     }
     
     @FXML
-    void patientLookup(ActionEvent event){
-        
+    void patientLookup(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.PATIENT_LOOKUP);
+        AppSession.cm.showScreen(LookUp.PATIENT_REGISTRATION); 
     }
     
     @FXML
-    void doctorLookup(ActionEvent event){
-        
+    void doctorLookup(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.DOCTOR_LOOKUP);
+        AppSession.cm.showScreen(LookUp.DOCTOR_LOOKUP); 
     }
     
     @FXML
-    void userLookup(ActionEvent event){
-        
+    void userLookup(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.USER_LOOKUP);
+        AppSession.cm.showScreen(LookUp.USER_LOOKUP);
     }
     
     @FXML
-    void myPatients(ActionEvent event){
-        
+    void myPatients(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.MY_PATIENT);
+        AppSession.cm.showScreen(LookUp.MY_PATIENT);
     }
     
     @FXML
-    void myLabRequests(ActionEvent event){
-        
+    void myLabRequests(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.MY_LAB_REQUESTS);
+        AppSession.cm.showScreen(LookUp.MY_LAB_REQUESTS); 
     }
     
     @FXML
-    void registerNewUser(ActionEvent event){
-        
+    void registerNewUser(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.REGISTER_NEW_USER);
+        AppSession.cm.showScreen(LookUp.REGISTER_NEW_USER);
     }
     
     @FXML
-    void labRequests(ActionEvent event){
-        
+    void labRequests(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.LAB_REQUESTS);
+        AppSession.cm.showScreen(LookUp.LAB_REQUESTS);
     }
     
     @FXML
-    void log(ActionEvent event){
-        
+    void log(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.LOG);
+        AppSession.cm.showScreen(LookUp.LOG);
     }
     
     @FXML
-    void labPatients(ActionEvent event){
-        
+    void labPatients(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.LAB_PATIENT);
+        AppSession.cm.showScreen(LookUp.LAB_PATIENT);
     }
     
     @FXML
-    void redBloodCell(ActionEvent event){
-        
+    void redBloodCell(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.RED_BLOOD_CELLS);
+        AppSession.cm.showScreen(LookUp.RED_BLOOD_CELLS);
     }
     
     @FXML
-    void whiteBloodCell(ActionEvent event){
-        
+    void whiteBloodCell(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.WHITE_BLOOD_CELLS);
```

Comment:
you could have another method that calls both of these
it is excessive to call two methods with the same input every time.

Reply:
I don't know what you mean. I'm using whatever is provided for me with the FXML

Reply:
i mean like 
```java
AppSession.cm.loadAndShow(Lookup.XXX)
```